azkaban-common/src/main/java/azkaban/project/FlowTriggerSchedule.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.project;
+
+import com.google.common.base.Preconditions;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * FlowTriggerSchedule is an immutable class
+ * which holds all necessary properties of a schedule.
+ */
+public class FlowTriggerSchedule {
+
+  private final String type;
```

Comment:
Why not just cronExpression string? This is the only schedule we support today.

Reply:
I think it's better to let the schedule class be consistent with the definition of schedule config in the yaml file.